config/sql/migration/migrate.7.sql

```diff
@@ -0,0 +1,9 @@
+-- Don't modify this! Create a new migration instead--see docs/schema-migrations.adoc
+BEGIN TRANSACTION;
+
+CREATE TABLE installation_result(id TEXT, result_code INTEGER NOT NULL DEFAULT 0, result_text TEXT);
```

Comment:
The change to `schema.sql` is also missing

Reply:
Fixed. Sorry, forgot about this file. Should we maybe have an automated test that the final schema after running through the migrations matches the expected schema?